test/simapp/app.go

```diff
@@ -254,47 +288,58 @@ func NewSimApp(
 	// set the BaseApp's parameter store
 	app.ConsensusParamsKeeper = consensusparamkeeper.NewKeeper(
 		appCodec,
-		keys[consensusparamtypes.StoreKey],
+		runtime.NewKVStoreService(keys[consensusparamtypes.StoreKey]),
```

Comment:
use the `runtime.NewKVStoreService` method to create a `KVStoreService` from a `StoreKey`: